pyproject.toml

```diff
@@ -18,8 +18,8 @@ requires = [
   "wheel",
   "toml",
   "versioningit",
-  "numpy == 1.20.3",
-  "scipy == 1.9.0"
+  "numpy >= 1.20.3",
```

Comment:
This should probably be `>1.20<2.0` to prevent accidentally moving forward. It shouldn't be an issue with mantid being involved.